gcp-autocomplete-lib.js

```diff
@@ -6,8 +6,8 @@ const helpers = require("./helpers");
 function generateGCPAutocompleter(
   ClientClass,
   fieldsToSelect,
-  listingFunction = genericList,
   filterGenerator = null,
+  listingFunction = genericList,
 ) {
```

Comment:
Isn't listing function more likely to be provided than filter generator? I think previous order made more sense, however I genuinely don't know and would love to here your thoughts.